Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py

```diff
@@ -349,17 +349,31 @@ def get_user(user, properties):
 
 def list_users_command():
     properties = demisto.args().get('properties', 'id,displayName,jobTitle,mobilePhone,mail')
-    users_data = list_users(properties)
-
+    next_page = demisto.args().get('next_page', None)
+    users_data, result_next_page = list_users(properties, next_page)
     users_readable, users_outputs = parse_outputs(users_data)
     human_readable = tableToMarkdown(name='All Graph Users', t=users_readable, removeNull=True)
+
     outputs = {'MSGraphUser(val.ID == obj.ID)': users_outputs}
+    if result_next_page:
+        human_readable += "To get further results, enter this to the next_page parameter:\n" + str(result_next_page)
```

Comment:
Add a ScreenShot to the PR for how does it look in the warroom

Reply:
and maybe just add this info in the metadata of the tabletomarkdown

Reply:
done!